claim.md

```diff
@@ -0,0 +1,78 @@
+# 💬 Claim
+
+In this section, you'll play the role of a <span class="label-role claimer">claimer</span>:  
+* You'll make a claim about yourself;  
+* You'll request an attestation of your claim. You need to, because a claim in itself has no value. To become valid in the eyes of <span class="label-role verifier">verifiers</span>, it needs to be attested by a trusted entity.
```

Comment:
Wordy:  "You'll request an attestation of your claim. You need to because a claim in itself has no value. To become valid in the eyes of <span class="label-role verifier">verifiers</span>, it needs to be attested by a trusted entity."

I think this needs restructuring. It is super good, but I think it needs to be changed.

"A claim in itself has no value; you'll have to request an attestation of your claim.  For a claim to become valid in the eyes of verifiers, it needs to be attested by a trusted entity."